Ginger/Ginger/WindowExplorer/Java/JavaElementTreeItemBase.cs

```diff
@@ -34,17 +35,25 @@ public abstract class JavaElementTreeItemBase : TreeViewItemBase
         public Boolean IsExpandable { get { return this.JavaElementInfo.IsExpandable; } set { this.JavaElementInfo.IsExpandable = value; } }
 
        public List<ITreeViewItem> Childrens()
-        {         
-            PayLoad Response = getChilderns();
-            List<PayLoad> controls = Response.GetListPayLoad();
-            List<ITreeViewItem> Childrens= new List<ITreeViewItem>();
-            if (Response.Name == "ContainerComponents")           
-               Childrens = GetControlsAsTreeItems(controls);
-          
-           if (Response.Name == "HTML Element Children")
-               Childrens = GetHTMLControlsAsTreeItems(controls);
+        {
+            List<ITreeViewItem> Childrens = new List<ITreeViewItem>();
+            try
```

Comment:
try catch is fine. But do we know why we are getting the exception in the first place ?

Reply:
pushed changes for java side to avoid the exception